lib/task/migrate/arUpgradeSqlTask.class.php

```diff
@@ -246,6 +250,196 @@ protected function execute($arguments = [], $options = [])
+    {
+        foreach (['about', 'privacy'] as $slug) {
+            // Ignore missing static pages when no confirmation mode is on so it has to be approved manually
+            if (null === QubitStaticPage::getBySlug($slug)
+              && (!$options['no-confirmation']
+                && $this->askConfirmation(
+                [
+                    sprintf('The %s page has been deleted. Do you wish to recreate it? (y/N)', $slug),
+                ],
+                'QUESTION_LARGE',
+                false
+              ))
+            ) {
+                $page = new QubitStaticPage();
+                $page->slug = $slug;
+                $page->sourceCulture = 'en';
+                $page->save();
+            }
+        }
+    }
+
+    /**
+     * For all standard static pages, reset translations in the database
+     * to match the current AtoM default values.
+     */
+    protected function resetChangedStaticPages()
+    {
+        $changes = $this->getStaticPageTranslationResetValues();
+
+        foreach ($changes as $slug => $translationData) {
+            $page = QubitStaticPage::getBySlug($slug);
+
+            if (null !== $page) {
+                foreach ($translationData as $culture => $translations) {
+                    if (isset($translations['title'])) {
+                        $page->setTitle($translations['title'], ['culture' => $culture]);
+                    }
+
+                    if (isset($translations['content'])) {
+                        $page->setContent($translations['content'], ['culture' => $culture]);
+                    }
+                }
+            }
+
+            $page->save();
+        }
+    }
+
+    /**
+     * Generate an array containing the static page translation values needed
+     * to change translations in the database to the current AtoM default values.
+     *
+     * Example of the structure:
+     *
+     *  [
+     *    'someslug' => [
+     *      'en' => [
+     *        'title' => 'Default English Title'
+     *      ]
+     *    ],
+     *    'anotherslug' => [
+     *      'fr' => [
+     *        'content' => 'Contenu français par défaut'
+     *      ]
+     *    ]
+     *  ]
+     *
+     * @return array
+     */
+    protected function getStaticPageTranslationResetValues()
+    {
+        $changes = [];
+
+        foreach ($this->parseStaticPageFixtureData() as $slug => $translationData) {
+            $page = QubitStaticPage::getBySlug($slug);
+
+            // Only check if page still exists in the database (the privacy page can be deleted)
+            if (null !== $page) {
+                $pageChanges = [];
+
+                // Check each standard translation against the database version of it
+                foreach ($translationData as $culture => $translations) {
+                    $pageChanges[$culture] = [];
+
+                    // Check page title
+                    if ($translations['title'] != $page->getTitle(['culture' => $culture])) {
```

Comment:
This method structure is very similar to `resetChangedStaticPages()`; is there a reason to collect all of the changes in an array, rather than just resetting the values directly in `resetChangedStaticPages()`?